features/networking/multus.feature

```diff
@@ -416,4 +416,107 @@ Feature: Multus-CNI related scenarios
     Then the output should contain "net1"
     And the output should contain "net2"
     And the output should contain 2 times:
-      | macvlan mode bridge |
+	    | macvlan mode bridge |
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24488
+  @admin
+  Scenario: Create pod with Multus bridge CNI plugin without vlan
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
+    And the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project_name clipboard
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/bridge-host-local-novlan.yaml |
+      | n | <%= cb.project_name %>                                                                                               |
+    Then the step should succeed
+    #Creating no-vlan pod abosrbing above net-attach-def
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/pod-bridge-host-local-novlan.yaml |
+      | n | <%= cb.project_name %>                                                                                                   |
+    Then the step should succeed
+    And the pod named "pod-novlan" becomes ready
+    And evaluation of `pod` is stored in the :pod clipboard
+    When I execute on the pod:
+      | /usr/sbin/ip | -d | link |
+    Then the output should contain "net1"
+    #Entering into corresponding no eot make sure No VLAN ID information shown for secondary interface
+    Given CNI vlan info is obtained on the "<%= cb.pod.node_name %>" node
+    Then the step should succeed
+    And the output should contain 2 times:
+      | 1 PVID untagged |
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24489
+  @admin
+  Scenario: Create pod with Multus bridge CNI plugin and vlan tag
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
```

Comment:
This should be supported from 4.2

Reply:
I guess we shall remove such checks. Anyways when we run tests we select specific version in polarion and trigger on corresponding setup and polarion also has right version set. Not sure where such gates are useful.

Another view. oc version keeps breaking itself time to time like currently my client says "Client Version: openshift-clients-4.3.0-201909231341" which is wierd.

Reply:
The gating trend was there when 3.x and 4.x had same branch master. So may be its not useful now

Reply:
removing version gates for now unless anybody has any reason to keep them

Reply:
It is useful when you want to test some framework change. Such checks let you avoid running impossible tests for your cluster.
e.g. grepping where some step is used and running such scenarios to see if your changes did not break something.
I agree it is not super important but is definitely helpful.